pkg/riff/commands/credential_delete_test.go

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package commands_test
+
+import (
+	"github.com/projectriff/riff/pkg/riff/commands"
+	"github.com/projectriff/riff/pkg/testing"
+	"github.com/projectriff/system/pkg/apis/build"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestCredentialDeleteOptions(t *testing.T) {
+	table := testing.OptionsTable{
+		{
+			Name: "valid multi-delete",
+			Options: &commands.CredentialDeleteOptions{
+				DeleteOptions: testing.ValidDeleteOptions,
+			},
+			ShouldValidate: true,
+		},
+		{
+			Name: "invalid multi-delete",
+			Options: &commands.CredentialDeleteOptions{
+				DeleteOptions: testing.InvalidDeleteOptions,
+			},
+			ExpectFieldError: testing.InvalidDeleteOptionsFieldError,
+		},
+	}
+
+	table.Run(t)
+}
+
+func TestCredentialDeleteCommand(t *testing.T) {
+	credentialName := "test-credential"
+	credentialAltName := "test-alt-credential"
```

Comment:
To me, "alt" means "another name for the same thing". Shouldn't this be named `otherXXX` rather, or `additionalXXX`?

Reply:
yep

Reply:
done